tests/randomized/lib/EnvFileGenerator.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace RandomizedTests\Tooling;
+
+class EnvFileGenerator
+{
+    public function generate($destination, $scenarioName)
+    {
+        file_put_contents(
+            $destination,
+            \str_replace(
+                ['{{scenario_name}}'],
+                [$scenarioName],
+                file_get_contents(__DIR__ . '/templates/.env.template')
+            )
+        );
```

Comment:
This pattern appears a number of times. Perhaps there could be a generic template generator akin to the generic substitutions supported by `DockerComposeFileGenerator::generate()`?

Not a blocker for merge, but just a thought for later if you agree. :)

Reply:
This is a very good point. I merged this PR so we can provide value sooner in master, but I also open #1155 so we can iterate there over this feedback. Let me know what do you think and let's continue the conversation there.